docs/catalog/cloud/cache-static-data.md

```diff
@@ -15,12 +15,17 @@ tags:
 <PatternComponent></PatternComponent>
 
 ## Description
-From an energy-efficiency perspective, it's better to reduce network traffic by reading the data locally through a cache rather than accessing it remotely over the network. 
+Relying on local data caches instead of remote access significantly improves energy efficiency. 
+Shorter data journeys require less energy transmission, similar to how fewer network devices traversed translate to lower embodied carbon emissions.
 
-Shortening the distance a network packet travels means that less energy is required to transmit it. Similarly, from an embodied carbon perspective, we are more efficient with hardware when a network packet traverses through less computing equipment. 
+However, a cost analysis is key. We need to compare the energy cost of data transfer with the combined cost of local storage (including its embodied carbon footprint from increased memory usage).
+
+For frequently accessed static data, especially by multiple users, consider caching on a Content Delivery Network (CDN). This reduces individual data transfers per user request, minimizing overall energy consumption.
+
+If data is required multiple times for each user then caching locally might be advantageous after a propet benefit analysis.
 
 ## Solution
-Caching static data, such as images or audio, instead of transferring it over the network. 
+Caching static data, such as videos, images or audio, instead of transferring it over the network if the cost benefit analysis proves it being more efficient. 
```

Comment:
Caching static data, such as videos, images, or audio, instead of transferring it over the network if the cost-benefit analysis proves it being more efficient. 